cli/src/main/java/keywhiz/cli/ClientUtils.java

```diff
@@ -76,14 +76,17 @@
    * @param cookies list of cookies to include in the client.
```

Comment:
Add a `@param` entry for the Keystore and explain what happens when null is supplied.